src/GraphQL/Types/Substrate/TokenMetadataType.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+namespace Enjin\Platform\GraphQL\Types\Substrate;
+
+use Enjin\Platform\GraphQL\Types\Traits\InSubstrateSchema;
+use Enjin\Platform\Interfaces\PlatformGraphQlType;
+use Rebing\GraphQL\Support\Facades\GraphQL;
+use Rebing\GraphQL\Support\Type;
+
+class TokenMetadataType extends Type implements PlatformGraphQlType
+{
+    use InSubstrateSchema;
+
+    /**
+     * Get the type's attributes.
+     */
+    public function attributes(): array
+    {
+        return [
+            'name' => 'TokenMetadata',
```

Comment:
The fields on this type match up with the new `MetadataInputType`, are they specifically for tokens only?  If so, might it be worth renaming the `MetadataInputType` to `TokenMetadataInputType` for clarity?

Reply:
Correct, will do, thanks

Reply:
Done, renamed, yes, it is only for tokens.